alg/crc32c.c

```diff
@@ -63,7 +63,7 @@ init(void)
 
 	/* Fill in tables. */
 	for (i = 0; i < 256; i++) {
-		r = reverse(i);
+		r = reverse(i & 0xffff);
```

Comment:
In that other code, we were doing some arithmetic and then saying "take the last byte of this".  Here we're saying "this is really a 32-bit integer and has been all along".

Either way works (and they should produce identical code) but I like to have the "natural reading" of the code match my mental model of what we're trying to do, if that makes sense.